mon-pix/app/components/challenge-item-qcu.js

```diff
@@ -13,10 +11,11 @@ class ChallengeItemQcu extends ChallengeItemGeneric {
 
   // FIXME refactor this
   _getAnswerValue() {
-
-    return this.$('.challenge-proposals input:radio:checked').map(function() {
-      return this.getAttribute('data-value');
-    }).get().join('');
+    const checkedInputValues = [];
+    document.querySelectorAll('input[type="radio"]:checked').forEach(function(element) {
+      checkedInputValues.push(element.getAttribute('data-value'));
+    });
+    return checkedInputValues.join('');
```

Comment:
Je propose

```suggestion
    const checkedInputValues = [];
    const rawRadioInputs = document.querySelectorAll('input[type="radio"]:checked');
    rawRadioInputs.forEach(function(rawRadioInput) {
      checkedInputValues.push(rawRadioInput.getAttribute('data-value'));
    });
    return checkedInputValues.join('');
```

Et au passage supprimer le // FIXME